### PR TITLE
[MIG][16.0] mrp_order_report_product_barcode: Migration to 16.0

### PR DESCRIPTION
**Pull request by labour-angus**
Old PR: #129 (suspended)

### CODE REVIEW

**Comment by labour-angus** [comment]
@pedrobaeza I've had a few problems with the following command:
git filter-branch --tree-filter 'if [ -d mrp_order_report_product_barcode ]; then mv mrp_order_report_product_barcode mrp_order_report_lot_barcode; fi' HEAD
because it always closed my PRs (I've also tried on this PR #133 )
In the end I just edited it manually (no migration scripts ATM).

Could you explain to me what I did wrong?? I'll tell you what I've tried to do:
I've updated the local branch with the most recent updates on manufacture-reporting and:
1. cherry pick the mrp_order_report_product_barcode from 11.0
2. changed the module's name with the command
3. edited the files
4. pre commit + commited stuff
5. commit the edited files

With this procedure I was sure to reopen a succesfull PR without needing to rebase.
But instead I always got an error "There isn't anything to commit" and force pushing locally always gave me "Everything up-to-date" (this behaviour is what I've also experienced from when the original PR got automatically closed)

**Comment by pedrobaeza** [comment]
You need to do a rebase after the `git filter-branch` command. I put it on my instructions.

**Comment by labour-angus** [comment]
Ok I probably rebased on the old branch

**Comment by labour-angus** [comment]
@mymage Can I ask you to review this PR once more? Thank you in advance

**Review by matteotognini — APPROVED** [review]
functional ok

File: mrp_order_report_lot_barcode/README.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+==============================================
+Lot Barcode on the Production Order Report
+==============================================
```
Comment (pedrobaeza) [inline mrp_order_report_lot_barcode/README.rst]:
```suggestion
==========================================
Lot Barcode on the Production Order Report
==========================================
```

or you can fill the `readme` folder and let the bot to regenerate this file.

File: mrp_order_report_lot_barcode/README.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+Usage
+=====
+
+* Create a manufacturing order for the BOM created above
+* Print the Production Order
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+    :alt: Try me on Runbot
+    :target: https://runbot.odoo-community.org/runbot/131/11.0
```
Comment (pedrobaeza) [inline mrp_order_report_lot_barcode/README.rst]: This reference is totally obsolete

```suggestion
```

Comment (labour-angus) [reply mrp_order_report_lot_barcode/README.rst]: Oops I'll change it ASAP

**Review by mymage — APPROVED** [review]
LGTM

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Comment by labour-angus** [comment]
merge?

**Pull request merged**